pkg/drawio/squareTreeNode.go

```diff
@@ -206,11 +200,53 @@ func NewSubnetTreeNode(parent *ZoneTreeNode, name, cidr, acl string) *SubnetTree
 }
 
 func (tn *SubnetTreeNode) children() ([]SquareTreeNodeInterface, []IconTreeNodeInterface, []LineTreeNodeInterface) {
-	return []SquareTreeNodeInterface{}, tn.elements, tn.connections
+	return tn.groupSquares, tn.elements, tn.connections
 }
 func (tn *SubnetTreeNode) Label() string {
 	return labels2Table([]string{tn.name, tn.cidr, tn.acl})
 }
 func (tn *SubnetTreeNode) SetACL(acl string) {
 	tn.acl = acl
 }
+func (tn *SubnetTreeNode) NIs() []IconTreeNodeInterface {
+	nis := []IconTreeNodeInterface{}
+	for _, icon := range tn.elements {
+		if icon.IsNI() {
+			nis = append(nis, icon)
+		}
+	}
+	return nis
+}
+
+// /////////////////////////////////////////////////////////////////////////////////////
+type groupSquareVisibility int
+
+const (
+	theSubnet groupSquareVisibility = iota
+	square
+	innerSquare
+	connectedPoint
+)
+
+type GroupSquareTreeNode struct {
+	abstractSquareTreeNode
+	groupies   []IconTreeNodeInterface
+	visibility groupSquareVisibility
```

Comment:
Should we promote `visibility` to `abstractSquareTreeNode` and the respective getter/setter to `SquareTreeNodeInterface`? Will this help with subnet grouping?

Reply:
didnt figure out the subnet grouping yet.
I might use there GroupSquareTreeNode too, so visibility should stay here